lib/_http_client.js

```diff
@@ -653,6 +664,12 @@ function parserOnIncomingClient(res, shouldKeepAlive) {
       response: res,
     });
   }
+  if (isTraceHTTPCategory() && req._traceEventId) {
```

Comment:
I believe `typeof req._traceEventId !== 'undefined'` is a faster and more safe approach

Reply:
Thanks ! Done !